Update readme with bintray instructions (#575)
opentok/opentok-react-native: README.md

Newer versions of Android–`API Level 23` (Android 6.0)–have a different permissions model that is already handled by this library.

#### Bintray sunset

> Bintray support has ended (official announce: [https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)). On your app build.gradle file you need to remove reference to `jcenter` and replace it to `mavenCentral`. Example: 

Reviewer: ```suggestion
Bintray support has ended (official announcement: [https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)). In your app build.gradle file you need to remove reference to `jcenter` and replace it with `mavenCentral`. Example: 
```
